textproc/lilypond/Portfile

```diff
@@ -3,9 +3,9 @@
 PortSystem          1.0
 
 name                lilypond
-version             2.18.2
-revision            11
-set branch          [join [lrange [split ${version} .] 0 1] .]
+set branch          2.20
```

Comment:
Mhmm, I would prefer that if I open a `Portfile` I can immediately see the version.  With this your change I have to look at two lines...

Reply:
What about using `version 2.20.0`  and `set branch 2.20` and having them on adjacent lines? This way it would avoid the "complex and error-prone" approach of hard coding the branch in the `master_sites`, and it would make the version immediately apparent.

Reply:
What about deriving the branch from the version information, too?  Right now, this is possible...

Reply:
If people would really prefer to continue using `set branch [join [lrange [split ${version} .] 0 1] .]` then that's what we can do, I just find it to be unnecessarily opaque.

Reply:
Then please add a comment, say,
```tcl
# Compute the branch from the version number.
```

Reply:
Computing the branch from the version number is what is recommended in https://trac.macports.org/wiki/PortfileRecipes#branch. The goal is that someone updating the portfile only has to update the version line and the checksums line.